### PR TITLE
media-sound/vorbis-tools: multiple vulnerabilities fixed

### PR DESCRIPTION
**Pull request by ChrisADR**
Bug reports:

https://bugs.gentoo.org/show_bug.cgi?id=537422

### CODE REVIEW

File: media-sound/vorbis-tools/Manifest [hunk 1]
```diff
@@ -1 +1,12 @@
...
+MISC ChangeLog 2806 SHA256 2d8d60944fd614f156920d24de77149211138c9618b5a3685048881c515e132e SHA512 c6924898bfdae85aba5a99205cd9a038f16c7eeebee0eb8b67fc93308ab500011977a809b46e09644f69e95558e233ab88fded93579769d11b9e124a9dd6c4ad WHIRLPOOL f42ed83f4706140558dc17f8e2e27fcdb855eac57988a14571a9704cc186c37140e2384f56bb4020f52230cdb1b9215b858152f7805a4df370323af962922666
+MISC ChangeLog-2015 15500 SHA256 d856ad70db28a7c0c9cacb6cafe7ef74b574c51f80ac95e3c9144c3ab7225b99 SHA512 5f0f9dd1aa725444a0d26c6e21036d67f0f088e36d2366e7a313cc86551e590798bc54f9390f29055c6ba7bfa74f99fb6bd39658d299cc61156d11cb8c159ac7 WHIRLPOOL a9a2afc0751ffcde59ca3b7eeec036c8befe207124516c4d1f763c5d93634c3fe456c3e7acaf2cb4b66b5526fcdc9c7ba1e58c13d8de884a7731f9876eea2332
+MISC metadata.xml 427 SHA256 41126e7adad0e82b0b7b5a6c6b4bcd546f14700a80ecfbcdcd4dec0c9d1b0dae SHA512 fe38791f10b0312286c49042e5c24303418ab755d5300b64b7dc406be5dfda993bbe804eae1124c2484d6a1f2e823895e174a522f8710e4b12129e856674d65c WHIRLPOOL 06607c58e62cbe5b9fc9cab69e354c7afa7062f1b1704f3ad334e11eecaa61152194f350831ae133c0c47fb1f2703d3d48cca7e8abf7e18f834b4a0f016319a0
```
Comment (aballier) [inline media-sound/vorbis-tools/Manifest]: did you use repoman manifest?

File: media-sound/vorbis-tools/files/vorbis-tools-1.4.0-CVE-2014-9638.patch [hunk 2]
```diff
@@ -0,0 +1,92 @@
+Patches taken as references: 
+https://github.com/mark4o/opus-tools/commit/8c412e619b83eb6dd32191909cf6672e93e5802e
+https://trac.xiph.org/attachment/ticket/2212/0001-oggenc-Fix-large-alloca-on-bad-AIFF-input.patch
+To fix bug report:
+http://www.openwall.com/lists/oss-security/2015/08/29/1
+    https://bugs.gentoo.org/show_bug.cgi?id=559170
+https://bugs.gentoo.org/show_bug.cgi?id=537422
```
Comment (aballier) [inline media-sound/vorbis-tools/files/vorbis-tools-1.4.0-CVE-2014-9638.patch]: whats the difference with current vorbis-tools-1.4.0-aiff-buffer-overflow.patch ?

**Comment by ChrisADR** [comment]
It fixes a couple of vulnerabilities, and instead of using a separated patch for the aiff bufferoverflow, I combined all the patchs from that particular file (audio.c) in order to avoid possible  conflicts between patches

**Comment by Polynomial-C** [comment]
How about adding new ebuilds with EAPI-6?

**Comment by ChrisADR** [comment]
Well I suppose that I could try, right now is only a security matter, because I'm part of the security team and we have had those bugs for a while, but after closing them I could try to make a new ebuild with EAPI-6

**Comment by aballier** [comment]
merged, thx

**Pull request closed without merging**